src/ariel-os-coap/Cargo.toml

```diff
@@ -54,6 +56,9 @@ workspace = true
 # laze's name for this (where coap-server makes more sense).
 coap-server = []
 
+coap-server-config-unprotected = []
```

Comment:
Should I refer to the docs of other items, or do we understand "this is a feature documented in ariel-os or laze" to be a default assumption?